waiter/src/waiter/request_log.clj

```diff
@@ -75,7 +76,8 @@
       principal (assoc :principal principal)
       protocol (assoc :backend-protocol protocol)
       server (assoc :server server)
-      handle-request-latency-ns (assoc :handle-request-latency-ns handle-request-latency-ns))))
+      handle-request-latency-ns (assoc :handle-request-latency-ns handle-request-latency-ns)
+      (some? waiter-api-call?) (assoc :waiter-api waiter-api-call?))))
```

Comment:
Why is `some?` necessary?

Reply:
`some?` checks for non-nil value, else a `false` value would have been skipped here.